core/services/pipeline/task.median.go

```diff
@@ -21,6 +22,16 @@ func (t *MedianTask) Type() TaskType {
 	return TaskTypeMedian
 }
 
+func (t *MedianTask) ApplyDefaults(inputValues map[string]string, g TaskDAG, self taskDAGNode) error {
+	if _, exists := inputValues["allowedFaults"]; !exists {
+		if len(self.inputs()) == 0 {
+			return errors.Wrapf(ErrWrongInputCardinality, "MedianTask requires at least 1 input")
+		}
+		t.AllowedFaults = uint64(len(self.inputs()) - 1)
+	}
+	return nil
```

Comment:
so if its not set, then we tolerate 0 faults instead of 50% like before?

Reply:
No, if it's not set, we tolerate `n - 1` faults where `n` is the number of tasks feeding into the medianizer.